core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -128,7 +128,7 @@ class ZkMigrationIntegrationTest {
 
     val underlying = clusterInstance.asInstanceOf[ZkClusterInstance].getUnderlying()
     val zkClient = underlying.zkClient
-    val migrationClient = ZkMigrationClient(zkClient, PasswordEncoder.noop())
+    val migrationClient = ZkMigrationClient(zkClient, PasswordEncoder.noop(), MetadataVersion.latest())
```

Comment:
nit: Same as `replicaBrokerIds`, `MetadataVersion.latest` has no side-effect so it should be called without empty parentheses.